src/core/main.cpp

```diff
@@ -161,6 +161,7 @@ void printHelp()
 		"Actions:\n"
 		"  <no action> [options...] [<project>]  Start LMMS in normal GUI mode\n"
 		"  dump <in>                             Dump XML of compressed file <in>\n"
+		"  compress <in>                         Compress file <n>\n"
```

Comment:
I think this needs improving. `compress <in> > <out>` is more like an ordinary use case. I don't know if that's good syntax too but you get the idea. Also I don't know what it looks like on non linux/unix systems.

`Compress file <n>\n"`

Did you mean `Compress file <in>\n"`